lib/logflare_web/live/alerts/components/run_query_result.heex

```diff
@@ -0,0 +1,7 @@
+<div :if={@query_result_rows}>
```

Comment:
wrong file name, should be lib/logflare_web/live/alerts/components/run_query_result.html.heex